service/controller/resource/terminateunhealthynode/metrics.go

```diff
@@ -0,0 +1,32 @@
+package terminateunhealthynode
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	gaugeValue float64 = 1
+)
+
+var (
+	nodeAutoRepairTermination = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "azure_operator_unhealthy_node_termination",
+			Help: "Gauge representing node termination due to node auto repair feature.",
+		},
+		[]string{"cluster_id", "terminated_node", "terminated_instance_id"},
+	)
+)
```

Comment:
IIUC this will add a new metric with value `1` whenever a new node is terminated, and it will be there until the operator is restarted/terminated? or until when?